manager/src/main/java/com/redhat/service/smartevents/manager/workers/resources/ConnectorWorker.java

```diff
@@ -193,4 +193,14 @@ protected ConnectorEntity doDeleteDependencies(ConnectorEntity connectorEntity)
         return connectorEntity;
     }
 
+    private static boolean isSourceConnector(ConnectorEntity connectorEntity) {
+        return connectorEntity.getConnectorType().contains("_source_");
```

Comment:
Can we use a proper `type` check instead of a contains?

Reply:
Done :+1: